feat: add qa pr template (#131)
across-protocol/frontend-v2: pull_request_template.md

<!--
If you need QA on your PR include the following section in your code. This will alert the QA team
that they should be testing.  As an engineer use github emojis to :heart: :+1: if the QA comments are productive

> or :-1: if the QA comments are hindering productivity.

Author: I kind of leave this up to the dev or others to decide if responses are productive or unproductive. I imagine most comments would be, but maybe theres a better way to describe it. Basically if the contributions are useful and in good faith id consider it productive and vice versa for being unproductive.

> or :-1: if the QA comments are hindering productivity.

Author: I'll add some more clarification here

-->

## QA

**Overview**

<!--
Give context around the change, anything that would help the testers understand the issue, feature and the changes made.
Don't expect them to look at the code, but anything you think they should know before testing. Screenshots are
useful here to illustrate changes or new parts of the app.

Example:
This PR adds ability to speed up your transfers through the transfer tab. <screenshot of speedup button>
This change only affects the transfer tab, so do not expect changes in the app elsewhere.
-->

**Acceptance Criteria**

> <!--

Author: Yes very possible. They would need to delete the QA section from the pr details

Try to describe what the tester should be testing and what the expected behavior is. Enumerate them if possible

> so that testers can respond to the numbers directly if there are issues.

Author: i think we could strengthen the language for now. I'm afraid that it may be tedious or unecessary to add this, but we can see how it goes

Example:
1. See that you can send a transfer and then speed it up.  
2. See that transfer shows a speedup transaction hash after speeding it up.
3. See that there were no regressions with the transfer table.
-->

**How to test**
<!--
If particular parts of the app need testing, or you need specific actions to be taken, browsers to be used
please direct the testers on how to test the changes here. 

Example:
Use the generated kovan link and first send a transaction between any chains, then go to transfer tab.
You should see your active transfer, and the new speedup button.

> -->

Reviewer: How is this section differentiated from the acceptance criteria? Do the instructions here apply to _all_ acceptance criteria cases? Should we collapse this section into that one and allow for engineers to give general instructions that apply to all AC test cases?

Author: i guess the ACs are more about what to observe, and how to test is more about how to get the app to the state where you can observe the ACs.  We could probably put this section in the description for the ACs

### For QA Testers

> **Reporting QA Issues**

Reviewer: This section is the only one where the instructions aren't comments. Is this because we want these instructions to be read by testers and not engineers? If so, it might makes sense to add a quick comment here to tell engineers not to remove the text in this section.

> As a tester use github comments to report issues with the PR. Use screenshots if relevant, you can also

Reviewer: ```suggestion
Testers should use github comments to report issues with the PR. Use screenshots if relevant, you can also
```

> check the developer console to see if there are any logs which might be related. Use the following template

Reviewer: Might be useful to include a link to instructions on how to open the developer console.

> when reporting a problem. You submit multiple comments if there are multiple issues.

Reviewer: ```suggestion
when reporting problems or that all tests pass. Please submit multiple comments if there are multiple issues.
```

Author: this section is just for reporting issues

Add any screenshots of the app or console logs if you think it would be useful.
-->

**Repro Steps**

<!--
If this issue was seen after taking particular steps, then note how to do it here.

Example:
1. Go to bridge tab and send a tx between any chains.
2. go to pool tab before going to transaction tab.
3. see that my pending transaction is missing.
-->

**Testing Environment**

<!--
Optionally include anything about your testing environment, ie phone, tablet, chrome or safari, if you used
metamask or a different wallet, etc. This will let other testers know if there are configurations they could
test differently from yours.
-->

```

**Reporting QA Passed**

If you tested and did not find any issues, you will need to signal to the developer that this passed your QA.
Do this by adding comments in the pr. You should communicate to both the developers and other testers what you did during testing. Use the following
template to signal that QA passed your testing.

```
**QA Pass**

<!--
Describe how you tested, if you tested beyond what was originally specified, or any other parts of the app.

> Sometimes changes can cause what are called "regressions" (something was working, and now doesnt after this change)
> and its good to know that none happened even if outside of the scope of the original QA testing.

Reviewer: ```suggestion
Sometimes changes can cause what are called "regressions" (something was working, and now doesn't after this change).
This is important to report even if it's not part of the requested set of QA test cases.
```

Reviewer: @daywiss should we have a separate section for regressions? Should we tell them to report a QA issue even if it's a regression? Could putting it here lead to an engineer accidentally ignoring it? It feels odd to bury regressions in QA Passed, but not strongly opinionated.

Example:
Tested sending eth and weth between optimism and arbitrum on kovan and back again. Both cases i was able to see my 
transaction and speed it up. No issues with the UI but i wasnt sure if the transaction actually was faster.

-->

**Testing Environment**

<!--

> Optionally include anything about your testing environment, ie phone, tablet, chrome or safari, if you used

Reviewer: ```suggestion
Optionally include anything unique about your testing environment, ie phone, tablet, chrome or safari, if you used
```
